### PR TITLE
some simple gamelab eyes tests

### PR DESCRIPTION
**Pull request by Bjvanminnen**

### CODE REVIEW

File: apps/src/gamelab/AnimationTab/NewListItem.jsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ var NewListItem = React.createClass({
   render: function () {
     var hovered = Radium.getState(this.state, 'main', ':hover');
     return (
-      <div style={styles.tile} onClick={this.props.onClick}>
+      <div style={styles.tile} onClick={this.props.onClick} id="newListItem">
```
Comment (islemaster) [inline apps/src/gamelab/AnimationTab/NewListItem.jsx]: There can be more than one `NewListItem` on the page (in fact there is right now - one in the 'animations' column and one in the 'frames' column).  Should either pass an id to this component and make it unique, or give it a className instead.

File: dashboard/test/ui/features/gamelab/eyes.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@eyes
+@dashboard_db_access
+@as_student
```
Comment (islemaster) [inline dashboard/test/ui/features/gamelab/eyes.feature:3]: May need as_admin at some point in the near future, when we lock down gamelab again.  I'm working on this, could make this change myself when it's necessary.

**Comment by islemaster** [comment]
One id nit.  Otherwise LGTM.

**Comment by Bjvanminnen** [comment]
Realized I also wanted one outside of the context of projects. I'm going to copy a gamelab level into allthethings now, and then once it makes it to staging in the next push, will add an eyes test for it

**Pull request merged**